src/components/ScenesHome/CourseCarousel.jsx

```diff
@@ -0,0 +1,106 @@
+import { useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { Button } from "../Button/Button";
+import "./Courses.css";
+
+export const CourseCarousel = () => {
+    const { t, ready } = useTranslation();
+
+    const coursesItems = t("home.courses", {returnObjects: true})
+
+    const coursesArray = t("home.courses.coursesArray", {returnObjects: true})
+
+    if (!ready) return "loading translations...";
```

Comment:
This is a great functionality of i18next that I had no idea existed, thanks for enlightening me!